content/en/tracing/connect_logs_and_traces/dotnet.md

```diff
@@ -17,11 +17,15 @@ further_reading:
       text: 'Correlate request logs with traces automatically'
 ---
 
-## Automatic Trace ID injection
+## Automatic log injection
```

Comment:
I'm a little concerned that this makes it sounds like we are "injecting logs," when actually we are injecting data into existing log. Is this the same wording we use elsewhere?